### PR TITLE
Mention golang_x_xerrors in README

### LINKED ISSUE #41: Add note regarding org_golang_x_xerrors to README
I think this is a mandatory step to use `gazelle_cabal` right now, if I'm not mistaken, so it should probably be noted in the README.

### PR DESCRIPTION
**Pull request by googleson78**
Fixes #41

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,16 @@ stack_snapshot(
...
 ```
+Right now there are also some issues with `gazelle` and as a temporary fix you need to include the following in your `WORKSPACE`:
+```python
+go_repository(
+    name = "org_golang_x_xerrors",
+    importpath = "golang.org/x/xerrors",
+    sum = "h1:go1bK/D/BFZV2I8cIQd1NKEZ+0owSTG1fDTci4IqFcE=",
+    version = "v0.0.0-20200804184101-5ec99f83aff1",
+)
+```
```
Comment (facundominguez) [inline README.md:58]: Should this appear before calling `gazelle_dependencies()`?

Comment (googleson78) [reply README.md:58]: Seems to work both before and after `gazelle_dependencies()`. Additionally, would it be possible to somehow temporarily include this in `gazelle_cabal_dependencies`? :thinking:

Comment (facundominguez) [reply README.md:58]: It could interfere if the user wanted to use a different version of `golang_x_xerrors`, I guess.

Comment (googleson78) [reply README.md:58]: I guess we can just merge this as-is then?

**Review by facundominguez — APPROVED** [review]

**Comment by facundominguez** [comment]
Merging. Thanks @googleson78!

**Pull request merged**